### PR TITLE
Add initial post on Offsite 2019

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: offsite-2019.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Goals
+-----
+
+Our biggest goal was building a shared undertanding of the vision for the project.
+Being fully remote,
+it's often hard to communicate at a deep level about mission and vision.
+Being together in person for a week gave us the space to understand each other and our views better.
+
+The other biggest goal was to build a Roadmap for the next 3-6 months for the project.
```
Comment (agjohnson) [inline offsite-2019.rst]: Wild title case

```suggestion
The other biggest goal was to build a roadmap for the next 3-6 months for the project.
```

File: offsite-2019.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    Before, it felt like we were 5 contributors working on an open source project. Now, it feels like we're members of a team.
+
+For each member of the team,
+there was someone else they had never met in person before this.
+Having all met in person will make it much easier to collaborate online going foward,
+and to feel like we are working towards a goal that we all share.
+
+In terms of technical output,
+the Roadmap we have established will make us much more productive in our work going foward.
```
Comment (agjohnson) [inline offsite-2019.rst]:
```suggestion
the roadmap we have established will make us much more productive in our work going foward.
```

File: offsite-2019.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Format
+------
+
+We used the following format for the project:
+
+* Before the offsite, we created a Trello board with a list of topics to discuss. This was all the major feature ideas, issues, and concerns that came up over the previous couple months.
+* For the first 3 days, we went through the list and discussed each topic. We took notes (almost 20 pages) of these discussions, and mostly tried to build a shared understanding on a path forward.
+* The 4th day we broke each section down into action items, and chunked them into 1-3 smaller sections of work. We called these "v1", "v2", and "v3" to break out the stages where each project could be shipped and have impact on our users.
+* The 5th day we took all the tasks, then sized and prioritized them. We broke things out into 4 piles representing months going forward, and tried to balance the tasks so each month had a similar amount of work in it. We then turned this into a Trello Roadmap board.
```
Comment (agjohnson) [inline offsite-2019.rst]:
```suggestion
* The 5th day we took all the tasks, then sized and prioritized them. We broke things out into 4 piles representing months going forward, and tried to balance the tasks so each month had a similar amount of work in it. We then turned this into a Trello roadmap board.
```

File: offsite-2019.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+and to feel like we are working towards a goal that we all share.
+
+In terms of technical output,
+the Roadmap we have established will make us much more productive in our work going foward.
+The entire team now has vision around the roadmap items,
+and understands the tasks other people are working on much better.
+This has already lead to a much better ability to collaborate together.
+
+We are hoping to do another offsite in 2020,
+and if we do our jobs right hopefully we'll have another teammate or two.
```
Comment (agjohnson) [inline offsite-2019.rst]:
```suggestion
and if we do our jobs right, hopefully we'll have another teammate or two.
```

File: offsite-2019.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+* David Fischer
+* Manuel Kaufmann
+* Eric Holscher
+* Anthony Johnson
+* Santos Gallegos
+
+Goals
+-----
+
+Our biggest goal was building a shared undertanding of the vision for the project.
```
Comment (stsewd) [inline offsite-2019.rst]:
```suggestion
Our biggest goal was building a shared understanding of the vision for the project.
```

File: offsite-2019.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+and to feel like we are working towards a goal that we all share.
+
+In terms of technical output,
+the roadmap we have established will make us much more productive in our work going foward.
+The entire team now has vision around the roadmap items,
+and understands the tasks other people are working on much better.
+This has already lead to a much better ability to collaborate together.
+
+We are hoping to do another offsite in 2020,
+and if we do our jobs right, hopefully we'll have another teammate or two.
```
Comment (davidfischer) [inline offsite-2019.rst]: :+1:

**Comment by ericholscher** [comment]
Cool, I will go ahead and merge it today then 👍

**Pull request merged**